### PR TITLE
Improve a little overriding module page

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.x
| Description?  | Improve a little overriding module page
| Fixed ticket? |

### CODE REVIEW

File: themes/reference/overriding-modules.md [hunk 1]
```diff
@@ -97,7 +98,7 @@ The issue goes both ways: if you want to modify the `included-template.tpl` file
 {include file='./included-template.tpl'}
 ```
 
-PrestaShop 1.7 introduces a new cool way to include files in module templates. By using this method, all the expected rules will be followed:
+PrestaShop 1.7 introduces a new way to include files in module templates. By using this method, all the expected rules will be followed:
```
Comment (matks) [inline themes/reference/overriding-modules.md:101]: "Cool" is not something I personally expect from a documentation for developers.

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
Cool, thanks @matks  😛

**Comment by eternoendless** [comment]
Does this need to be backported?

**Comment by matks** [comment]
> Does this need to be backported?

No because `8.x` branch content is different

**Pull request merged**